### PR TITLE
add ernie-skep (chnsenticorp)

### PR DESCRIPTION
**Pull request by Steffy-zxf**
add module ernie_skep_sentiment_analysis 

SKEP: Sentiment Knowledge Enhanced Pre-training for Sentiment Analysis. Ernie_skep_sentiment_analysis module is initialize with enie_1.0_chn_large when pretraining. This module is finetuned on ChnSentiCorp dataset to do sentiment claasification. It can do sentiment analysis prediction directly, label as positive or negative.

### CODE REVIEW

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
+# coding:utf-8
+# Copyright (c) 2019  PaddlePaddle Authors. All Rights Reserved.
```
Comment (nepeplwu) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: 2020

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/model/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# -*- coding:utf-8 -**
```
Comment (nepeplwu) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/model/__init__.py]: 空文件就不加LICENSE了

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+        return pooled_output, sequence_output
+
+    def array2tensor(self, arr_data):
+        """
+        convert numpy array to PaddleTensor
+        """
+        tensor_data = PaddleTensor(arr_data)
+        return tensor_data
+
+    def classify_sentiment(self, texts=[], use_gpu=False):
```
Comment (nepeplwu) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: 接口和senta保持一致

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+        Returns:
+            res (list): The result of sentiment label and probabilties.
+        """
+
+        if use_gpu:
+            try:
+                _places = os.environ["CUDA_VISIBLE_DEVICES"]
+                int(_places[0])
+            except:
+                raise RuntimeError(
```
Comment (nepeplwu) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: 术语和其他module保持一致吧

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py (hunk 1 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: 已修改

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/model/__init__.py (hunk 2 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/model/__init__.py]: done

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py (hunk 3 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: function 已动词打头命名，已说明。

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py (hunk 4 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/module.py]: done

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+## 命令行预测
+
+```shell
+$ hub run ernie_skep_sentiment_analysis --input_text='虽然小明很努力，但是他还是没有考100分'
+```
+
+## API
+
+```python
+def classify_sentiment(texts=[], use_gpu=False)
```
Comment (ZeyuChen) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: classify_sentiment ->
sentiment_classify(xxx)

Comment (ZeyuChen) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: I think classify is not a mainstream words used in interface design
How about change classify_sentiment to sentiment_predict(xxx)

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+```python
+import paddlehub as hub
+
+# Load ernie_skep_sentiment_analysis module.
+module = hub.Module(name="ernie_skep_sentiment_analysis")
+
+# Predict sentiment label
+test_texts = ['你不是不聪明，而是不认真', '虽然小明很努力，但是他还是没有考100分']
+results = module.classify_sentiment(test_texts, use_gpu=False)
```
Comment (ZeyuChen) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: change interface name to sentiment_predict

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+# Get feature and main program of ernie_skep_sentiment_analysis
```
Comment (ZeyuChen) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: It would be better to separate python code block to make our function clear.

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+position_ids = inputs["position_ids"]
+segment_ids = inputs["segment_ids"]
+input_mask = inputs["input_mask"]
+
+# Use "pooled_output" for sentence-level output.
+pooled_output = outputs["pooled_output"]
+
+# Use "sequence_output" for token-level output.
+sequence_output = outputs["sequence_output"]
+
```
Comment (ZeyuChen) [inline hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: The code block here seems a little bit complicated, not easy to understand.

**Review by ZeyuChen — CHANGES_REQUESTED** [review]
Change interface name.
Separate code block to make our module function easier to understand.

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md (hunk 5 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: Done

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md (hunk 6 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: Done

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md (hunk 7 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: Done

File: hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md (hunk 8 above)
Comment (Steffy-zxf) [reply hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md]: Done
